convert_gvf_to_vcf/convertGVFtoVCF.py

```diff
@@ -207,34 +209,29 @@ def gvf_features_to_vcf_objects(gvf_lines_obj_list,
         if vcf_object.key in vcf_data_lines:
             vcf_data_lines[vcf_object.key].append(vcf_object)
         else:
-            vcf_data_line_objects_list = []
-            vcf_data_line_objects_list.append(vcf_object)
+            vcf_data_line_objects_list = [vcf_object]
             vcf_data_lines[vcf_object.key] = vcf_data_line_objects_list
         # check the number of objects to see if they are merged
         # for key in vcf_data_lines.keys():
         #     vcf_obj_list = vcf_data_lines[key]
-            # print("for ", key, " the number of vcf objects is: ", len(vcf_obj_list))
+        #     print("for", key, " the number of vcf objects is: ", len(vcf_obj_list))
     return header_standard_lines_dictionary, vcf_data_lines, list_of_vcf_objects
 
-
-def populate_sample_formats(list_of_sample_names):
-    """ Populates a dictionary using a list of sample names. Dictionary key is sample name, value is the sample's format value.
-    :param list_of_sample_names: list of sample names
-    :return:sample_name_format_value: dictionary of sample names => sample format value
-    """
-    sample_name_format_value = {}
-    for sample in list_of_sample_names:
-        sample_name_format_value[sample] = "sampleFORMAThere" #TODO: fill this in
-    return sample_name_format_value
-
-def format_sample_values(sample_name_format_value):
+def format_sample_values(sample_name_dict_format_kv, list_of_sample_names):
     """ Creates a partial vcf data line of sample format values.
-    :param sample_name_format_value: dictionary of sample names => sample format value
+    :param sample_name_dict_format_kv: dictionary of sample names => sample format value
+    :param list_of_sample_names: list of sample names
     :return: sample_format_values_string: formatted string
     """
     sample_format_values_string = ""
-    for key in sample_name_format_value:
-        sample_format_values_string = sample_format_values_string + sample_name_format_value[key] + "\t"
+    uniq_sample_name = list(dict.fromkeys(list_of_sample_names))
```

Comment:
To unique a list you should use a set
```suggestion
    uniq_sample_name = list(set(list_of_sample_names))
```
Although, I'm not sure why you're assuming that the list would contain duplicates.